CourseApp/Ship.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace CourseApp
+{
+    public abstract class Ship
```

Comment:
As for task - we should add at least couple of methods

Reply:
Additionally - let's not change this one for this PR (Remove this file for now and concentrate on first calculate task